torch/testing/_internal/common_methods_invocations.py

```diff
@@ -277,7 +280,22 @@ def get_operator_variant(self):
         """
         return self.operator_variant
 
-    def sample_inputs(self, device, dtype, requires_grad=False, **kwargs):
+    def conjugated_sample_inputs(self, samples):
+        """Returns an iterable of conjugated SampleInputs.
+        """
+
+        conj_samples = list(samples)
+
+        for i in range(len(samples)):
+            sample = conj_samples[i]
+            if isinstance(sample.input, torch.Tensor):
+                sample.input = sample.input.conj()
+            else:
```

Comment:
Comment here that the assumption is that the input is either a tensor or a sequence of tensors